dev-libs/libofx/libofx-0.9.14.ebuild

```diff
@@ -53,4 +56,5 @@ src_configure() {
 src_install() {
 	default
 	find "${D}" -name '*.la' -delete || die
+	use static-libs || ( rm "${ED}"/usr/$(get_libdir)/${PN}.a || die )
```

Comment:
Unnecessary subshell.
```
if ! use static-libs; then
	find "${D}" -name '*.a' -delete || die
fi
```

Reply:
I was a bit lazy there and wanted to keep it one line. I'm gonna change it.